src/module.ts

```diff
@@ -26,6 +27,7 @@ export default defineNuxtModule<ModuleOptions>({
     componentDirs: [],
     components: [],
     silent: true,
+    exclude: ['nuxt/dist/app/components/client-only', 'nuxt/dist/app/components/dev-only'],
```

Comment:
We may also want to exclude `RouteLocationRaw` from `VueRouter`
https://github.com/nuxt/framework/blob/main/packages/nuxt/src/app/components/nuxt-link.ts#L26-L27 which takes a lot to resolve

Reply:
Using theses gave me good results:
```js
componentMeta: {
    checkerOptions: {
      forceUseTs: true,
      schema: {
        ignore: [
          'ComponentData', // nuxt content
          'NuxtComponentMetaNames', // avoid loop
          'RouteLocationRaw', // vue router
          'RouteLocationPathRaw', // vue router
          'RouteLocationNamedRaw', // vue router
        ],
      },
    },
  },
```

Reply:
Hey :) thanks, I added these in [c9f7614](https://github.com/nuxtlabs/nuxt-component-meta/pull/47/commits/c9f7614fde30aed1c6fd6a1e8dcce3320d919890)